src/EditorFeatures/CSharpTest/Classification/SemanticClassifierTests.cs

```diff
@@ -763,7 +772,8 @@ void M()
                 Class("T"),
                 Class("T"),
                 Class("T"),
-                Field("field"));
+                Field("field"),
+                Static("field"));
```

Comment:
i personally don't ike that the test is like this.  i'd prefer that this be more like Static_Field("field") or Static(Field("field").  But i don't care hugely enough to force you to chance this.